pymatgen/analysis/diffusion/utils/parse_entries.py

```diff
@@ -185,7 +185,7 @@ def get_inserted_on_base(
     """
     mapped_result = get_matched_structure_mapping(base_ent.structure, inserted_ent.structure, sm)
     if mapped_result is None:
-        return None
+        raise ValueError("Cannot obtain inverse mapping, consider lowering tolerances in StructureMatcher")
```

Comment:
I think this is the only point where the behaviour of the code is changes. I hope this doesn't break anything.